data/hai/hai news.txt

```diff
@@ -165,3 +165,131 @@ news "reformed pirate in Hai space"
 			"The Hai will pay you to rush cargo to human space and back. Don't die trying."
 		word
 			`"`
+
+news "unfettered hai dialog"
+	location
+		government "Hai (Unfettered)"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Unfettered warlord"
+			"Hai youth"
+			"Hai warlord"
+	message
+		word
+			`"`
+		word
+			"How did you get here? You should've been blasted to bits by our ships."
+			"I came here because this is the only place in the galaxy where I feel wanted."
+			"Weak human. Why do you dare to come to the land of the true Hai?"
+			"Altered Hai are soft and weak-willed. They care not for true struggle or challenge."
```

Comment:
The fact that there is a mythos story about the Hai being Altered probably shouldn't be used for naming. Hai are able to defect back and forth with relative ease and all Unfettered were part of the greater whole until recently (long after the supposed altering), so Fettered is probably better, implying an ideological limit rather than an actual physical limit.
```suggestion
			"Fettered Hai are soft and weak-willed. They care not for true struggle or challenge."
```